.github/workflows/prod_test.yml

```diff
@@ -0,0 +1,37 @@
+name: End-to-end Production Testing (Nightly)
+on:
+  schedule:
+    # Run every morning Pacific Time.
+    # Random hour and minute to avoid creating excess traffic during popular times.
+    # Because the test takes a long time (> 30 min) to run, it is configured to run only once a day.
+    - cron:  '17 17 * * *'
+
+jobs:
+  build_and_test:
+    runs-on: [ubuntu-18.04, macos-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        target_arch: [aarch64, armhf]
+        target_os: [ubuntu]
+        rosdistro: [dashing]
+        install_type:
+          - pip install -e .
+          - pip install -i https://test.pypi.org/simple/ cross-compile
```

Comment:
note the new name ros_cross_compile (i don't think `cross-compile` with hyphens was correct either)

Reply:
I took this from what I found in test-pypi already. Renaming.